lms/static/sass/discussion/_layouts.scss

```diff
@@ -31,4 +31,8 @@
 
 .discussion-column {
     min-height: 500px;
+
+    .new-post-article {
```

Comment:
Yep, it does need the same fix:

![image](https://cloud.githubusercontent.com/assets/5985072/18144974/52c71334-6f97-11e6-8049-4f73743289c0.png)